Source/WebCore/platform/mac/ThemeMac.mm

```diff
@@ -949,6 +950,13 @@ static void paintColorWell(ControlStates& controlStates, GraphicsContext& contex
     UNUSED_PARAM(useSystemAppearance);
     UNUSED_PARAM(pageScaleFactor);
 
+#if HAVE(GPU_AVAILABILITY_CHECK)
+    if (!isGPUAvailable()) {
+        RELEASE_LOG_FAULT(Painting, "GPU is not available.");
+        return;
+    }
+#endif
```

Comment:
Early return seems better than crashing -- but do we have a plan to debug this further? Or is this considered normal for some reason?

Reply:
That is a good point! This is not considered normal, and the current theory is that Metal initialization is failing and we fall back to OpenGL rendering, which is not available. I have reached out to a reproducer of this issue, and requested more diagnostics, in order to better understand the root cause.

Thanks for reviewing!

Reply:
Can we call `isGPUAvailable` when IOKit is blocked? It consults Metal devices.

Reply:
> Can we call `isGPUAvailable` when IOKit is blocked? It consults Metal devices.

That is a good point, and will not work if called in the WebContent process.

Thanks for reviewing!